### ISSUE TITLE
Debugger Step/Continue not responding

### ISSUE THREAD

**Issue by noambonnie** [microsoft/vscode-go#2827]
My environment stopped responding to Step/Continue in the debugger. The breakpoint breaks, but neither commands work and only remedy is to stop the process, effectively rendering the debugger useless.

The only changes I can think of that happened are:
1. Upgrade to vscode`1.39` (upgraded to `1.39.1` - problem persists).
2. Upgrade of Xcode to `11.1` (don't think it's related but pointing it out since this is another thing that changed in my environment)

vscode-go version `0.11.7`

Working on MacOS Mojave `10.14.6` (Upgraded to Catalina - problem persists)

`go version go1.12.4 darwin/amd64`

**Steps to reproduce**
1. Set a breakpoint.
2. Try to continue.

**Comment by noambonnie** [comment]
Attaching trace logs - no logs are generated when stepping/continuing.
[vscode-go-debug.txt](https://github.com/microsoft/vscode-go/files/3718220/vscode-go-debug.txt)

**Comment by noambonnie** [comment]
```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "trace": "verbose",
            "program": "${workspaceFolder}/imageToBase64.go",
            "env": {},
            "args": []
        }
    ]
}
```

**Comment by noambonnie** [comment]
This seems to be resolved with 1.39.2. Closing...
